book/controller.rst

```diff
@@ -778,6 +778,20 @@ There are also special classes to make certain kinds of responses easier:
   :class:`Symfony\\Component\\HttpFoundation\\StreamedResponse`.
   See :ref:`streaming-response`.
 
+JSON helper
+~~~~~~~~~~~
+
+You can simplify sending JSON response with
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::json`
+helper::
```

Comment:
The greatest feature of this shortcut is the usage of the Symfony serializer if available. We surely have to document that imo